nms/app/packages/magmalte/app/views/traffic/PolicyFlows.js

```diff
@@ -66,6 +66,9 @@ const useStyles = makeStyles(() => ({
   removeIcon: {alignSelf: 'baseline'},
   dialog: {height: '640px'},
   title: {textAlign: 'center', margin: 'auto', marginLeft: '0px'},
+  description: {
+    color: colors.primary.mirage,
+  },
```

Comment:
I've seen this reused a few times across components. Is there a common place where we can put this?